packages/auth/src/accessToken/routes.ts

```diff
@@ -119,20 +120,25 @@ async function rotateToken(
     newToken = await deps.accessTokenService.rotate(ctx.accessToken.id, trx)
 
     if (!newToken) {
-      ctx.throw()
+      throw new Error('invalid access token')
     }
 
     accessItems = await deps.accessService.getByGrant(newToken.grantId, trx)
 
     await trx.commit()
   } catch (error) {
     await trx.rollback()
-    const errorMessage = 'Could not rotate token'
+    const errorMessage =
+      error instanceof Error ? error.message : 'Could not rotate token'
     deps.logger.error(
       { err: error instanceof Error && error.message },
       errorMessage
     )
-    ctx.throw(400, { message: errorMessage })
+    ctx.throw(
+      400,
+      GNAPErrorCode.InvalidRotation,
+      generateGNAPErrorResponse(GNAPErrorCode.InvalidRotation, errorMessage)
+    )
```

Comment:
```suggestion
    throwGNAPError(ctx, 400, GNAPErrorCode.InvalidRotation, message)
```
what do you think of something like this also? then we wouldn't need to define the `GNAPErrorCode.X` value twice

Reply:
I like it, looks cleaner that way too